src/cmd-import

```diff
@@ -231,10 +231,12 @@ def skopeo_inspect(image):
 def parse_timestamp(timestamp):
     # datetime's doesn't support nanoseconds.
     # So trim it.
-    if len(timestamp) > 26 and timestamp[19] == '.':
-        timestamp = timestamp[:26] + "Z"
+    # ignore the microseconds too, as not all images has it.
+    timestamp, _, _ = timestamp.partition('.')
+    if not timestamp.endswith('Z'):
+        timestamp += 'Z'
 
-    timestamp = datetime.datetime.strptime(timestamp, '%Y-%m-%dT%H:%M:%S.%fZ')
+    timestamp = datetime.datetime.strptime(timestamp, '%Y-%m-%dT%H:%M:%SZ')
```

Comment:
How about replacing all this with `timestamp = dateutil.parser.parse()`?

(It's kinda funny because we're parsing a timestamp which must be RFC3339 by the OCI spec, to only reserialize it as another RFC3339 timestamp. We could just pass it through, but I agree the sanity-check roundtrip is helpful.)